### ISSUE TITLE
Will not run on Windows 10 1903 b18362.418

### ISSUE THREAD

**Issue by GuiltyPixel** [IJHack/QtPass#486]
Whether installed via the installer release on GitHub, or through Choco, versions 1.3.2, 1.3.1, and 1.3.0 fail to start with no error, nothing in the event viewer, no log to disk, no console output, no output of any kind. I get no tray icon or indication of a crash, it seems to simply exit silently for no identifiable reason.

I have Kleopatra from the gpg4win bundle installed, version 3.1.8 on gpg4win 3.1.10. They've been tested and work fine. Also, PWGEN 2.9.0 is installed. Windows is pretty much stock Windows 10 Professional 1903 build 18362.418 except I have enabled BitLocker without a TPM in the GPE, and disabled Cortana and Web Search functions via their reqpective registry keys. Nothing else has been messed with yet, it's just stock OOB Windows installed 3 days ago and updated.

**Comment by annejan** [comment]
This is a know issue and should be fixed in #482 https://github.com/IJHack/QtPass/releases/tag/v1.3.2

I'll give the new release a go on a clean Win10 VM too (tested on Win7)

**Comment by annejan** [comment]
First attempt, installing and running on Windows 10 Enterprise 170928-1534
![Screenshot 2019-10-10 at 17 51 36](https://user-images.githubusercontent.com/294470/66585250-cbde4f80-eb86-11e9-80a8-609c83c5b72c.png)
No issues . . also trayicon and everything works fine . . 

Don't worry . . more tests incoming . .

**Comment by annejan** [comment]
![Screenshot 2019-10-10 at 18 05 38](https://user-images.githubusercontent.com/294470/66586308-af431700-eb88-11e9-8623-18e21fb89a36.png)

Currently setting up 3 VMs . . 2 for testing and one for Development 🕺

**Comment by GuiltyPixel** [comment]
Tried 1.3.2 3 times, didn't work. Today I try it again, it works... No new installed libraries or applications and no Windows Update, but it works? I did restart between prior attempts... It's got plenty of problems, such as it failed to detect gpg on startup but does detect it when I hit `Autodetect` under `Configuration>Programs`. Does not detect default location of PWGEN either way. Sometimes browses to `/` instead of `/Users/<user>/passwordstore/`, installer needs to be run as admin since you forgot to add that flag to the installer executable... There is work to be done but it is functional. I wish I could explain how to reproduce. Closing.
